### PR TITLE
Document replicating a Cloud organization

### LINKED ISSUE #3067: Guide to migrate/copy data in InfluxDB Cloud
Multiple InfluxDB Cloud users have asked how to replicate the state of an organization. For example, copying a dev setup to a new organization. This can be done by using a combination of InfluxDB templates to replicate the metadata of an organization metadata (dashboards, buckets, etc) and CLI commands to copying data.

We should add a guide on how to do this in InfluxDB Cloud and the v2 API. , i.e. creating a complete copy of both metadata+data. The content would likely be applicable for a similar OSS guide as well.

There are multiple ways to copy data so the guide should outline the different approaches and when to use them.

1. Export query results as CSV, then write/upload the CSV into a new bucket using the CLI or curl.
    - Most flexible. Requires copying data to location outside of InfluxDB Cloud.
2. Write results directly to the new bucket with Flux `to()`.
    - May be limited based on the size of data. May encounter auth complexity copying data across organizations.

Rate limits should also be mentioned.

The guide should also cover the use of InfluxDB templates (or stack) to replicate metadata (dashboards, buckets, etc) before copying the data, i.e. creating a complete copy of both metadata+data. Can link to the [relevant template documentation](https://docs.influxdata.com/influxdb/cloud/influxdb-templates/create/).

##### Relevant URLs
This would probably be best placed in the section on [managing organizations](https://docs.influxdata.com/influxdb/cloud/organizations/) or managing buckets.

### PR DESCRIPTION
**Pull request by pierwill**
Closes #3067.

Also adds an example of using `--rate-limit` flag with `influx write`.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   3. Write the CSV data into a bucket in the new organization
+      using the [`influx write`](/influxdb/cloud/reference/cli/influx/write/) command.
+   
+### Write data with Flux
+Perform query for desired data.
+Write results directly to a bucket in the new organization with the Flux
+[`to()` function](http://localhost:1313/flux/v0.x/stdlib/influxdata/influxdb/to/).
+   
+   {{% note %}}
+Rate limiting may apply.
```
Comment (gunnaraasen) [inline content/influxdb/cloud/organizations/migrate-org.md:46]: Do we have a doc we could link about how to identify rate limit errors?

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+seotitle: Migrate organization in InfluxDB
+description: Replicate the state of an organization in InfluxDB
+menu:
+  influxdb_cloud:
+    name: Migrate organization
+    parent: Manage organizations
+weight: 110
+---
+
+To duplicate the state of an organization:
```
Comment (gunnaraasen) [inline content/influxdb/cloud/organizations/migrate-org.md:12]: I think a quick summary explanation would help set context for the following steps. E.g. an org's state consists of both metadata resources (dashboards, buckets, etc) and the data (time-series).

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+   2. Save the results as CSV.
+      (This requires copying data to a location outside of InfluxDB Cloud.)
+   3. Write the CSV data into a bucket in the new organization
+      using the [`influx write`](/influxdb/cloud/reference/cli/influx/write/) command.
+   
+### Write data with Flux
+Perform query for desired data.
+Write results directly to a bucket in the new organization with the Flux
+[`to()` function](http://localhost:1313/flux/v0.x/stdlib/influxdata/influxdb/to/).
+
```
Comment (gunnaraasen) [inline content/influxdb/cloud/organizations/migrate-org.md:44]: Might want to add a note that points to a migration doc for InfluxDB v1 to v2.

**Review by gunnaraasen — COMMENTED** [review]
Just a few small comments, otherwise this looks great and exactly what I had in mind.

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+menu:
+  influxdb_cloud:
+    name: Migrate organization
+    parent: Manage organizations
+weight: 110
+---
+
+The state of an consists of metadata (dashboards, buckets, and other resources) and data (time-series).
+While there no way to automatically duplicate an organization,
+the state of an organization can be migrated to a new organization.
```
Comment (gunnaraasen) [inline content/influxdb/cloud/organizations/migrate-org.md:14]: I'm not sure this is correct. Migrating the entire metadata and data state of an organization is effectively duplicating the org. Maybe something along the lines of:

An organization's state at a point in time can be effectively duplicated to another organization by copying both metadata and data. Copying only metadata can be useful for replicating the state of an organization for different environments, e.g. dev and prod. Copying only data can be useful for saving high value data to a location not subject to a retention policy.

**Review by gunnaraasen — CHANGES_REQUESTED** [review]
Looking good. Almost there.

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+   to the new organization.
+3. **Migrate data**.  
+   Use one of the methods below to copy data to the new organization:
+
+   - [Export data to CSV](#export-data-to-csv)
+   - [Write data with Flux](#write-data-with-flux)
+4. Re-invite users.
+
+### Export data to CSV
+1. Perform a query to return all desired data.
```
Comment (kelseiv) [inline content/influxdb/cloud/organizations/migrate-org.md:33]:
```suggestion
1. Perform a query to return all specified data.
```

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+2. Save the results as CSV.
+   (This requires copying data to a location outside of InfluxDB Cloud.)
```
Comment (kelseiv) [inline content/influxdb/cloud/organizations/migrate-org.md:35]:
```suggestion
2. Save the results as CSV (to a location outside of InfluxDB Cloud.)
```

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+3. Write the CSV data into a bucket in the new organization
+   using the [`influx write`](/influxdb/cloud/reference/cli/influx/write/) command.
+
+### Write data with Flux
+Perform a query to return all desired data.
```
Comment (kelseiv) [inline content/influxdb/cloud/organizations/migrate-org.md:40]:
```suggestion
Perform a query to return all specified data.
```

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+Write results directly to a bucket in the new organization with the Flux
+[`to()` function](/flux/v0.x/stdlib/influxdata/influxdb/to/).
+
+{{% note %}}
+The manual processes above may be affected by rate limiting.
+Use the [`influx write --rate-limit`](/influxdb/cloud/reference/cli/influx/write/#write-annotated-csv-data-using-rate-limiting)
+flag to control the rate of writes.
```
Comment (kelseiv) [inline content/influxdb/cloud/organizations/migrate-org.md:47]:
```suggestion
If writes are prevented by rate limiting, use the [`influx write --rate-limit`](/influxdb/cloud/reference/cli/influx/write/#write-annotated-csv-data-using-rate-limiting) flag to control the rate of writes.
```

File: content/influxdb/cloud/organizations/migrate-org.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+---
+title: Migrate an organization
+seotitle: Migrate an organization in InfluxDB Cloud
+description: Replicate the state of an organization in InfluxDB Cloud
```
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/influxdb/cloud/organizations/migrate-org.md:4]: Think we should use the most accurate term, either migrate, duplicate, or replicate; and only use one term to reduce cognitive load (unless there's a distinction we're making here).

**Review by kelseiv — APPROVED** [review]

**Pull request merged**